Freshli.Test/Unit/Python/VersionMatcherTest.cs

```diff
@@ -256,68 +273,67 @@ string secondChildBaseVersion
     [InlineData("1.0.0", ">0.0.1", true)]
     [InlineData("1.0.0", ">0.1.0", true)]
     [InlineData("1.0.0", ">1.0.0", false)]
-    [InlineData("1.0.0", ">1.0.0-alpha", true)]
+    [InlineData("1.0.0", ">1.0.0a1", true)]
     [InlineData("1.0.0", ">1.0.1", false)]
     [InlineData("1.0.0", ">1.1.0", false)]
     [InlineData("1.0.0", ">2.0.0", false)]
     [InlineData("1.0.0", ">=0.0.1", true)]
     [InlineData("1.0.0", ">=0.1.0", true)]
     [InlineData("1.0.0", ">=1.0.0", true)]
-    [InlineData("1.0.0", ">=1.0.0-alpha", true)]
+    [InlineData("1.0.0", ">=1.0.0a1", true)]
     [InlineData("1.0.0", ">=1.0.1", false)]
     [InlineData("1.0.0", ">=1.1.0", false)]
     [InlineData("1.0.0", ">=2.0.0", false)]
     [InlineData("1.0.0", "~=0.0.1", false)]
     [InlineData("1.0.0", "~=0.1.0", false)]
-    [InlineData("1.0.0", "~=1", true)]
```

Comment:
Per [PEP-440](https://www.python.org/dev/peps/pep-0440/#compatible-release), `~=1` should never be used:

> This operator MUST NOT be used with a single segment version number such as ~=1.